db/redis.go

```diff
@@ -379,9 +388,7 @@ func (r *RedisDriver) InsertCpes(fetchType models.FetchType, cpes models.Fetched
 						delete(oldDeps["VP"], vendorProductStr)
 					}
 				}
-				_ = pipe.SAdd(ctx, titleListKey, c.Title)
 				newDeps["Titles"][c.Title] = map[string]struct{}{}
```

Comment:
"Titles" in newDeps is used in maps.Keys() below.
So, the value here can be just struct{}{} ?

----

My two cents:
The field newDeps["Titles"] is not stored in redis. 
Making a new map[string]struct{} variable "titles" (and removing "Titles" filed from newDeps) may show what is stored in redis as DEP clearly ?

Reply:
> "Titles" in newDeps is used in maps.Keys() below.
So, the value here can be just struct{}{} ?

cannot use struct{}{} (value of type struct{}) as map[string]struct{} value in assignment
https://go.dev/play/p/RrdhuztnE4b

Reply:
> My two cents:
The field newDeps["Titles"] is not stored in redis.
Making a new map[string]struct{} variable "titles" (and removing "Titles" filed from newDeps) may show what is stored in redis as DEP clearly ?

https://github.com/vulsio/go-cpe-dictionary/compare/82b51ab3fa2074c38df329f589288ccffe603835..a1b7351fdd881074f7004df4e84111619e28edbc#diff-47497c0e5aee216ce040118072f4e2199cc00ec8f2c55059b2c4729ec026e101R350